cmd/operator-sdk/build/cmd.go

```diff
@@ -91,8 +92,8 @@ func createBuildCommand(imageBuilder, context, dockerFile, image string, imageBu
 }
```

Comment:
We need to add the flag with its description to the command and check if the specific `-skip-image` flag was used instead of a number of args > 0

E.g (something as):
```
cmd.Flags().BoolVar(&c.skipImage, "skip-image", false,"If true, will skip the image built with Docker")
```